guides/common/modules/proc_increasing-the-logging-level-for-redis.adoc

```diff
@@ -0,0 +1,27 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="increasing-the-logging-level-for-redis"]
+= Increasing the logging level for Redis
+
+You can find the log for Redis in `/var/log/redis/redis.log`.
+
+.Procedure
+. In `/etc/redis.conf`, set the logging level to `debug`:
```

Comment:
This isn't quite correct. Debian has always used `/etc/redis/redis.conf` while EL9 also adopted that (prior versions used `/etc/redis.conf`).
```suggestion
. In `/etc/redis/redis.conf`, set the logging level to `debug`:
```